src/Serialization/main.jl

```diff
@@ -341,22 +341,3 @@ include("Fields.jl")
 include("ToricGeometry.jl")
 include("Rings.jl")
 include("polymake.jl")
-
-
-@deprecate save_cone(Obj::Cone, filename::String) save(Obj, filename)
-@deprecate load_cone(filename::String) load(Obj, filename)
```

Comment:
Clearly these deprecated load methods were also wrong and would result in an error if someone actually used them